src/responder/nss/nss_protocol_grent.c

```diff
@@ -402,9 +402,13 @@ sss_nss_protocol_fill_initgr(struct sss_nss_ctx *nss_ctx,
     ret = sysdb_search_group_by_origgid(NULL, domain, orig_gid, NULL,
                                         &primary_group_msg);
     if (ret != EOK) {
-        DEBUG((ret == ENOENT ? SSSDBG_FUNC_DATA : SSSDBG_MINOR_FAILURE),
-              "Unable to find primary gid [%d]: %s\n",
-              ret, sss_strerror(ret));
+        if (ret == ENOENT) {
+            DEBUG(SSSDBG_FUNC_DATA, "There is no override for this group\n");
+        } else {
+            DEBUG(SSSDBG_MINOR_FAILURE,
+                  "Unable to find primary gid [%d]: %s\n",
```

Comment:
Well, I specially want to avoid the word "unable." It gives the felling that there is something that SSSD should be doing but it can't, and that after failing to do this thing, things may go wrong.

On the other side, "Unable to find the original group id attribute" is almost the same as what we had before. Do you have some other proposal?

Reply:
Hi,

I meant to replace the `else` message `Unable to find primary gid`. What I want to try to avoid is `primary gid` because a primary gid is mandatory in POSIX and what we are looking for is something different.

About `Unable`, in the `else` branch there is really some issue because the search should, if everything goes well, only return `EOK` or `ENOENT` so there is something which " SSSD should be doing but it can't". To make the consequences more clear, what about "Unable to find the original group id attribute, assuming there is none" for the `else` message?

bye,
Sumit

Reply:
I see. That is a good idea.